bundle/run/pipeline_options.go

```diff
@@ -0,0 +1,68 @@
+package run
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/databricks/cli/bundle/config/resources"
+	"github.com/databricks/databricks-sdk-go/service/pipelines"
+	flag "github.com/spf13/pflag"
+)
+
+// PipelineOptions defines options for running a pipeline update.
+type PipelineOptions struct {
+	// Perform a full graph update.
+	RefreshAll bool
+
+	// List of tables to update.
+	Refresh []string
+
+	// Perform a full graph reset and recompute.
+	FullRefreshAll bool
+
+	// List of tables to reset and recompute.
+	FullRefresh []string
+}
+
+func (o *PipelineOptions) Define(fs *flag.FlagSet) {
+	fs.BoolVar(&o.RefreshAll, "refresh-all", false, "Perform a full graph update.")
```

Comment:
The name of flags are very unintuitive but I guess they are just in line with API naming?

Reply:
Indeed. Aside from API, this is also in line with how these are named in the web UI.,